ls1intum/Artemis
# `Development`: Update server dependencies

Keep server dependencies up to date

### Checklist
#### General




### Steps for Testing
1. Navigate through the most important pages in Artemis and check that nothing is broken


#### Code Review
#### Manual Tests
